src/main/java/formflow/library/email/EmailClient.java

```diff
@@ -3,8 +3,15 @@
 import java.io.File;
 import java.util.List;
 
+/**
+ * Requires sendEmail methods for any class implementing EmailClient.
+ * @param <T> A Generic that can be set as the return type for the sendEmail method
+ */
 public interface EmailClient<T> {
 
+  /**
+   * A boolean variable required for any EmailClient
+   */
   Boolean requireTls = true;
```

Comment:
This inspired conversation about EmailClient being an interface and default scope for this variable.  When there is a variable in an Interface, it is by default "public Static final", which is obviously what we don't want here.  We are creating a ticket to rethink the logic here.

Reply:
https://www.pivotaltracker.com/story/show/186251813